lib/js-api/test/content-services/searchAiApi.spec.ts

```diff
@@ -113,4 +105,19 @@ describe('SearchAiApi', () => {
             });
         });
     });
+
+    describe('getConfig', () => {
+        it('should load knowledge retrieval configuration', (done) => {
+            searchAiMock.mockGetConfig200Response();
```

Comment:
mock?

Reply:
Yes, this is correct. We mock response from http request to don't call http for unit tests as it would not be good approach :) We should call http for e2e tests but not for unit tests.